src/ol/dom/dom.js

```diff
@@ -298,3 +298,51 @@ ol.dom.transformElement2D =
     // content size.
   }
 };
+
+
+/**
+ * Get the current coordinates of the given element relative to the document.
+ * Equivalent to jQuery's `$(el).offset()`.
+ * @param {!Element} element Element.
+ * @return {ol.Pixel}
+ */
+ol.dom.offset = function(element) {
+  var rect = element.getBoundingClientRect();
+
+  return [
+    rect.left + goog.global.document.body.scrollLeft,
+    rect.top + goog.global.document.body.scrollTop
+  ];
+};
```

Comment:
Closure Library includes a `goog.style.getPageOffset` function that we could use instead of introducing our own. But `goog.style.getPageOffset` uses `goog.style.getBoundingClientRect_`, which includes code dealing with legacy browsers (IE < 9).

We support IE 8 when `ol.LEGACY_IE_SUPPORT` is `true`, so theoretically, the overlay auto pan functionality should work correctly on that browser.

I am not sure. What do you think?